bcda/responseutils/v2/writer.go

```diff
@@ -161,7 +161,9 @@ func CreateOpOutcome(severity fhircodes.IssueSeverityCode_Value, code fhircodes.
 }
 
 func WriteError(outcome *fhirmodelOO.OperationOutcome, w http.ResponseWriter, code int) {
-	w.Header().Set(constants.ContentType, constants.JsonContentType)
+	//Write application/fhir+json header on OperationOutcome responses
+	//https://build.fhir.org/ig/HL7/bulk-data/export.html#response---error-status-1
+	w.Header().Set(constants.ContentType, constants.FHIRJsonContentType)
```

Comment:
What are the implications of updating the header for our consumers? Just wondering if there a compatibility between `JsonContentType` and `FHIRJsonContentType` or if our consumers would need to make any code changes on their end.

Reply:
They _shouldn't_ need to make any changes, unless they've adopted a practice of expecting OperationOutcomes to be accompanied by Content-Type=application/json. Since the CapabilityStatement from BCDA includes application/json and application/fhir+json, I'd imagine the risk of introducing an error in this manner is very low.